wrangler-core/src/main/java/io/cdap/directives/parser/ParseLog.java

```diff
@@ -89,10 +89,16 @@ public void destroy() {
           log = (String) object;
         } else if (object instanceof byte[]) {
           log = new String((byte[]) object);
+        } else if (object == null) {
+          throw new DirectiveExecutionException(
+            String.format("Error encountered while executing '%s' : Column '%s' has null value. " +
+                            "It should be of type 'String' or 'byte array'.", NAME, column)
```

Comment:
A String can be null, so the message can be a little confusing.

be of type -> be a non-null